CNKI.js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "5c95b67b-41c5-4f55-b71a-48d5d7183063",
 	"label": "CNKI",
 	"creator": "Aurimas Vinckevicius",
-	"target": "^https?://([^/]+\\.)?cnki\\.net",
+	"target": "^?://([^/]+\\.)?cnki\\.net",
```

Comment:
This needs to be reverted to its original value.

Reply:
OK, I found many errors from the CI check result. Do I need to fix all the errors?

Reply:
You don't have to fix linting errors. We can do that as well. However, it is difficult now to see, which functional changed you did now. Are your first two commits the functional changes and all the remaining ones for fixing linting issues according to Travis?

Reply:
Hi @zuphilip  Sorry about that.  The changes:
1. added return value in `getIDFromURL`
2. added new function `getIDFromRef`, `getPDF`, `getCAJ` and `getAttachments` 
3. Change code in  `getIDFromPage` to get `dbname` and `filename` pre-released article.
4. Change code in `scrape`, fix `CN` field error  and add attachments.